core/flyout.js

```diff
@@ -845,6 +846,20 @@ Blockly.Flyout.prototype.addBlockListeners_ = function(root, block, rect) {
 };
 
 /**
+ * Actions to take when a block in the flyout is right-clicked.
+ * @param {!Event} e Event that triggered the right-click.  Could originate from
+ *     a long-press in a touch environment.
+ * @param {Blockly.BlockSvg} block The block that was clicked.
+ */
+Blockly.Flyout.blockRightClick_ = function(e, block) {
```

Comment:
Yeah. I guess I just need to get used to paying closer attention to the definitions of these methods - it is super easy to miss that it isn't defined on the prototype and sadly js won't warn you when you try to use instance variables.